### DISCUSSION TITLE
Draw construction grids both on the water's surface and underwater

### DISCUSSION THREAD

**Discussion by HtotheTML** [OpenRCT2/OpenRCT2#21184] [Feature requests] [answered]
When building rides over water, the grid showing the ride's location draws underwater. In the same way that it was done for the land & construction rights tool in #21150 for #18632, it would be helpful if the drawn grid displayed on both the water's surface and underwater.

**Comment by Gymnasiast** [comment, marked as answer]
https://github.com/OpenRCT2/OpenRCT2/pull/21266
